Topology/Spectral/basic.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2024 Panagiotis Angelinos. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Panagiotis Angelinos
+-/
+
+import Mathlib.Topology.QuasiSeparated
+import Mathlib.Topology.Compactness.Compact
+import Mathlib.Topology.ContinuousOn
+import Mathlib.Topology.Sober
+
+/-!
+# Spectral Spaces
+This file defines spectral spaces. A topological space is spectral if it is sober,
+quasi-compact, the intersection of two quasi-compact opens is quasi-compact, and
+the collection of quasi-compact opens forms a basis for the topology.
+
+## Main Results
+
+- `SpectralSpace` : Predicate for a topological space to be spectral.
+- `compact_openEmbedding_of_SpectralSpace_is_SpectralSpace` : proof that a compact open subspace 
+    of a spectral space is spectral
+
+## Notation
+
+- None so far.
```

Comment:
Do you expect to add notation in the future? Otherwise I would just leave out this section.